blog/__init__.py

```diff
@@ -0,0 +1 @@
+"""Do not write anything here."""
```

Comment:
This is not a docstring as it doesn't describe the module's purpose. In fact, it's not even required anymore for Python 3 envs.